src/main/core/main.c

```diff
@@ -59,6 +59,22 @@ static void _main_logEnvironment(gchar** argv, gchar** envv) {
     }
 }
 
+static int _raise_rlimit(int resource) {
+    struct rlimit lim = {0};
+    if (getrlimit(resource, &lim) != 0) {
+        error("Could not get rlimit: %s", strerror(errno));
+        return -1;
+    }
+
+    lim.rlim_cur = lim.rlim_max;
+    if (setrlimit(resource, &lim) != 0) {
+        error("Could not update rlimit: %s", strerror(errno));
```

Comment:
Add limit type to the error message here too.

Reply:
Added.